### PR TITLE
Voting panel: improve the presentation of the status of a vote

### PR DESCRIPTION
**Pull request by bpierre**
Issue: https://github.com/aragon/aragon-apps/issues/464

### CODE REVIEW

**Comment by izqui** [comment]
A couple of issues I just detected:

- Token amounts formatting weirdly (token has 2 decimals, actual amounts are 1.03 and 1.00):
screen shot 2018-10-05 at 18 23 41

- `Vote not executed` label showing up even though the vote was executed, also `Vote will pass` shouldn't be shown when the vote has already passed:
screen shot 2018-10-05 at 18 24 00

**Comment by sohkai** [comment]
@izqui @bpierre @jounih Do you think the panel looks uneven if we just remove the subtext of the status?

image

File: apps/voting/app/src/components/VoteSuccess.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+import { theme, IconCross, IconCheck } from '@aragon/ui'
+
+const VoteSuccess = ({ success = true, vote }) => {
```
[the remark this answers is not in the document]
Comment (sohkai) [reply apps/voting/app/src/components/VoteSuccess.js]: See https://github.com/aragon/aragon-apps/pull/482 (ee28c8f)

File: apps/voting/app/src/components/VotePanelContent.js [hunk 2]
```diff
@@ -69,15 +71,14 @@ class VotePanelContent extends React.Component {
...
-        tokenContract.decimals()
-      )
+      tokenContract
+        .balanceOfAt(user, vote.data.snapshotBlock)
         .first()
-        .subscribe(([balance, decimals]) => {
+        .subscribe(balance => {
           const adjustedBalance = Math.floor(
-            parseInt(balance, 10) / Math.pow(10, decimals)
+            parseInt(balance, 10) / Math.pow(10, tokenDecimals)
```
[the remark this answers is not in the document]
Comment (sohkai) [reply apps/voting/app/src/components/VotePanelContent.js]: Actually nevermind, this isn't really important to use BN.js.

File: apps/voting/app/src/vote-utils.js [hunk 3]
```diff
@@ -14,25 +14,25 @@ export const EMPTY_CALLSCRIPT = '0x00000001'
...
+  return getVoteSuccess(vote) ? VOTE_STATUS_ACCEPTED : VOTE_STATUS_REJECTED
+}
+
+export function getVoteSuccess(vote) {
+  const { support, quorum } = vote
+  const { yea, nay } = vote.data
 
   const totalVotes = yea + nay
   const hasSupport = yea / totalVotes >= support
   const hasMinQuorum = getQuorumProgress(vote.data) >= quorum
```
[the remark this answers is not in the document]
Comment (sohkai) [reply apps/voting/app/src/vote-utils.js]: See https://github.com/aragon/aragon-apps/pull/482 (954c044).

File: apps/voting/app/src/components/SummaryRows.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  align-items: center;
+`
+
+const Bullet = styled.div`
+  flex-shrink: 0;
+  display: block;
+  width: 10px;
+  height: 10px;
+  margin-right: 15px;
+  border-radius: 5px;
```
[the remark this answers is not in the document]
Comment (sohkai) [reply apps/voting/app/src/components/SummaryRows.js]: See https://github.com/aragon/aragon-apps/pull/482 (954c044).

**Pull request merged**